mounts/org.osbuild.xfs

```diff
@@ -32,6 +32,11 @@ SCHEMA_2 = """
         "description": "mount the source as a readonly device",
         "type": "boolean",
         "default": false
+      },
+      "norecovery": {
+        "description": "mount the source as a readonly device",
```

Comment:
Description copied from readonly.

Reply:
> I wonder if it makes sense to always add `norecovery` (or `noload`) when doing `readonly` for filesystems that support it. From the ext4 docs:
> 
> > Mount filesystem read only. Note that ext4 will replay the journal (and thus write to the partition) even when mounted “read only”. The mount options “ro,noload” can be used to prevent writes to the filesystem.
> 
> There's a question here about what the `readonly` option means in the mount unit. Whether it maps directly to the `ro` mount option or if it should imply "read only and everything that should go with it".

That makes sense to me!
And in the option it's specified `readonly` not only `ro` which already can suggest it's not a 1:1 mapping.
So yeah let's do what you propose here!